app/jobs/clock/events_validation_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Clock
+  class EventsValidationJob < ApplicationJob
+    queue_as 'clock'
+
+    def perform
+      # NOTE: refresh the last hour events materialized view
+      Scenic.database.refresh_materialized_view(
+        Events::LastHourMv.table_name,
+        concurrently: false,
+        cascade: false
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)